tests/cacl/test_cacl_application.py

```diff
@@ -90,10 +90,8 @@ def docker_network(duthosts, enum_rand_one_per_hwsku_hostname, enum_frontend_asi
                       }
                      ]
     """
-    docker_network['bridge'] = {'IPv4Address': ipam_info['Config'][0].get('Gateway',
-                                                                          ipam_info['Config'][0].get('Subnet')),
-                                'IPv6Address': ipam_info['Config'][1].get('Gateway',
-                                                                          ipam_info['Config'][1].get('Subnet'))}
+    docker_network['bridge'] = {'IPv4Address' : ipam_info['Config'][0].get('Gateway', ipam_info['Config'][0].get('Subnet').split('/')[0]),
+                                'IPv6Address' : ipam_info['Config'][1].get('Gateway', ipam_info['Config'][1].get('Subnet').split('/')[0]+'1') }
```

Comment:
Maybe a short comment or snippet of output as background to clarify this parsing and splitting logic ?

Reply:
Updated comment and fixed ipv4 gateway ip derivation from subnet.